doc/reduce-traffic.md

```diff
@@ -23,7 +23,7 @@ longer serving historic blocks (blocks older than one week).
 Keep in mind that new nodes require other nodes that are willing to serve
 historic blocks.
 
-Whitelisted peers will never be disconnected, although their traffic counts for
+Peers with the `noban` permission will never be disconnected, although their traffic counts for
```

Comment:
Maybe underscores the relation that you may have hungry whitelisted peers provoking ban of honest peers and that something a node operator should care about ? "Resource control of whitelisted peers is left to the node operator or the whitelisting policy deployed"

Reply:
Agree but,

* There is the `TODO, exclude peers with noban permission`, so this will probably be fixed in the future by someone.
* I want to limit the changes in this pull request to a pure replacement of the word `Whitelist` to `noban permission`.